CMakeLists.txt

```diff
@@ -1,72 +1,24 @@
 cmake_minimum_required(VERSION 3.28)
-project(em)
+project(em
+  VERSION 1.0
+  DESCRIPTION "EmberScript - A small language for mathematical operations"
```

Comment:
Not [EmberScript](https://github.com/ghempton/ember-script)! :(

Reply:
ooops, fixing xd

Reply:
Addressed. Thanks for pointing it out.